apps/system/js/bluetooth_transfer.js

```diff
@@ -91,14 +98,14 @@ var BluetoothTransfer = {
       };
       req.onerror = function() {
         var msg = 'Can not check is device connected from adapter.';
-        self.debug(msg);
+        self._debug(msg);
         resolve(_('unknown-device'));
       };
     }.bind(this));
   },
 
-  debug: function bt_debug(msg) {
-    if (!this._debug) {
+  _debug: function bt__debug(msg) {
```

Comment:
Why is this change made instead of keeping debug()? Every other module in system app is using debug() instead of _debug()